django-stubs/contrib/auth/models.pyi

```diff
@@ -80,6 +83,9 @@ class AbstractUser(AbstractBaseUser, PermissionsMixin):
 
     EMAIL_FIELD: str
     USERNAME_FIELD: str
+
+    class Meta:
+        abstract: bool = True
```

Comment:
I've made the plugin code to look for `abstract = True`. But perhaps we should also look for `abstract: Literal[True]`?

Not sure what effect that has on/if inheriting e.g. 

```python
class Meta(AbstractUser.Meta):
    abstract = False
```

Reply:
I think we should support both.

Reply:
I've added support for both `abstract: Literal[True]` and `abstract = True` now